Doc/library/json.rst

```diff
@@ -283,6 +285,17 @@ Basic Usage
       input encoding should be UTF-8, UTF-16 or UTF-32.
 
 
+Constants
+^^^^^^^^^
+
+.. data:: compact
+
+   Constant that could be used as *separators* argument value
```

Comment:
`A constant that can be used as the *separators* argument to emit a compact serialization`.